Binärdateien luanti-5.11.0.orig/bin/luanti und luanti-5.11.0/bin/luanti sind verschieden.
Binärdateien luanti-5.11.0.orig/bin/minetest und luanti-5.11.0/bin/minetest sind verschieden.
diff -Nru luanti-5.11.0.orig/src/CMakeLists.txt luanti-5.11.0/src/CMakeLists.txt
--- luanti-5.11.0.orig/src/CMakeLists.txt	2025-02-14 19:38:27.000000000 +0100
+++ luanti-5.11.0/src/CMakeLists.txt	2025-02-16 21:25:08.841210820 +0100
@@ -692,6 +692,7 @@
 		sha256
 		${FREETYPE_LIBRARY}
 		${PLATFORM_LIBS}
+		socket
 		# on Android, Luanti depends on SDL2 directly
 		# on other platforms, only IrrlichtMt depends on SDL2
 		"$<$<PLATFORM_ID:Android>:${SDL2_LIBRARIES}>"
diff -Nru luanti-5.11.0.orig/src/gui/touchcontrols.cpp luanti-5.11.0/src/gui/touchcontrols.cpp
--- luanti-5.11.0.orig/src/gui/touchcontrols.cpp	2025-02-14 19:38:27.000000000 +0100
+++ luanti-5.11.0/src/gui/touchcontrols.cpp	2025-02-16 21:25:08.866232248 +0100
@@ -591,7 +591,7 @@
 					(!m_fixed_joystick && distance_sq > touch_threshold_sq)) {
 				m_joystick_has_really_moved = true;
 
-				m_joystick_direction = atan2(dir.X, -dir.Y);
+				m_joystick_direction = std::atan2(dir.X, -dir.Y);
 
 				const double distance = sqrt(distance_sq);
 				if (distance <= m_touchscreen_threshold) {
diff -Nru luanti-5.11.0.orig/src/porting.cpp luanti-5.11.0/src/porting.cpp
--- luanti-5.11.0.orig/src/porting.cpp	2025-02-14 19:38:27.000000000 +0100
+++ luanti-5.11.0/src/porting.cpp	2025-02-16 21:25:08.865345495 +0100
@@ -10,9 +10,11 @@
 
 #include "porting.h"
 
-#if defined(__FreeBSD__)  || defined(__NetBSD__) || defined(__DragonFly__) || defined(__OpenBSD__)
+#if defined(__FreeBSD__)  || defined(__NetBSD__) || defined(__DragonFly__) || defined(__OpenBSD__) || defined(__sun)
 	#include <sys/types.h>
+#ifndef __sun
 	#include <sys/sysctl.h>
+#endif
 	extern char **environ;
 #elif defined(_WIN32)
 	#include <windows.h>
diff -Nru luanti-5.11.0.orig/src/threading/thread.cpp luanti-5.11.0/src/threading/thread.cpp
--- luanti-5.11.0.orig/src/threading/thread.cpp	2025-02-14 19:38:27.000000000 +0100
+++ luanti-5.11.0/src/threading/thread.cpp	2025-02-16 21:25:08.850211140 +0100
@@ -295,7 +295,7 @@
 	return r == 0;
 #elif defined(__sun) || defined(sun)
 
-	return processor_bind(P_LWPID, P_MYID, proc_number, NULL) == 0
+	return processor_bind(P_LWPID, P_MYID, proc_number, NULL) == 0;
 
 #elif defined(_AIX)
 
--- luanti-5.12.0/src/skyparams.h.orig	2025-05-23 17:43:08.000000000 +0200
+++ luanti-5.12.0/src/skyparams.h	2025-06-15 12:10:12.298226003 +0200
@@ -4,6 +4,8 @@
 
 #pragma once
 
+#undef sun
+
 #include <vector>
 #include <string>
 #include "SColor.h"
